makefile

```diff
@@ -1,8 +1,21 @@
 install-dep:
 	pip install -r requirements.txt
-app-run:install-dep
-	flask run
-docker-build:
-	docker build -t studentapi:$(IMAGE_TAG) .
-docker-run: 
-	docker run -it -e FLASK_RUN_HOST=$(HOST) -e FLASK_RUN_PORT=$(PORT) -p $(PORT):$(PORT) studentapi:$(IMAGE_TAG)
+
+run:install-dep
+
+	flask -A student.py run --host=${HOSTPORT} --port=${APP_PORT}
+run-app:run
+
+#build application docker container.
+flaskapp_build:
+			docker build -t studentapi:1.0.0 .
```

Comment:
Why is Image tag hardcoded? This way you will never have a different tag for your image

Reply:
For this, I created env in github for passing image tag to makefile.